install-dev/upgrade/php/add_new_tab.php

```diff
@@ -53,12 +55,42 @@ function add_new_tab($className, $name, $id_parent, $returnId = false, $parentTa
 		');
     }
 
-    Db::getInstance()->execute('INSERT IGNORE INTO `'._DB_PREFIX_.'access` (`id_profile`, `id_tab`, `view`, `add`, `edit`, `delete`)
+    if (version_compare(_PS_VERSION_, '1.7.0.0', '<')) {
```

Comment:
why do we need to do that on a commit only available in 1.7.4+?

Reply:
What do you mean? This method is used by several methods in install-dev/upgrade/php.

Reply:
yes, but why do we do that? in which case you enter in the "if"?

Reply:
Agreed with @mickaelandrieu, you are pushing this change in `1.7.4.x`, so _PS_VERSION_ can't be `1.6` or something under `1.7` oO

Reply:
You never know from where you start the upgrade. `PS_VERSION` is not updated at each upgrade file. :wink: